ci/jenkins/test.sh

```diff
@@ -296,12 +296,8 @@ function clean_ns {
     matching_ns=$(kubectl get ns | awk -v ns="${ns}" '$1 ~ ns {print $1}')
     
     if [ -n "${matching_ns}" ]; then
-        echo "${matching_ns}" | while read -r ns_name; do
-            kubectl get pod -n "${ns_name}" --no-headers=true | awk '{print $1}' | while read pod_name; do
-                kubectl delete pod "${pod_name}" -n "${ns_name}" --force --grace-period 0
-            done
-            kubectl delete ns "${ns_name}" --ignore-not-found=true || true
-        done
+        kubectl delete all --all -n ${matching_ns} || true
```

Comment:
I understand that the previous code is not great, but I'm assuming there was a reason for `--force --grace-period 0`? Otherwise, why not just delete the Namespace, which will take care of deleting all resources in the Namespace?